CONTRIBUTING.md

```diff
@@ -0,0 +1,54 @@
+# Contribution guidelines
+
```

Comment:
Generally, I feel like `CONTRIBUTING.md` contains guidelines for contributing to the project itself (`in this case `CodemateLtd/cookbook`), but these guidelines go over some things not related to this project, such as GitLab and some additional references.

Maybe add an explainer for what these guidelines are for. If they're not for this project, maybe rename the file, to `project-guidelines.md` or something similar.

Reply:
I agree this is confusing but I'll keep it this way for now. There is too little content to justify the extra organisation.